docs/docs/01-ibc/06-channel-upgrades.md

```diff
@@ -164,3 +164,39 @@ IBC application callbacks should be primarily used to validate data fields and d
 > IBC applications should not attempt to process any packet data under the new conditions until after `OnChanUpgradeOpen`
 > has been executed, as up until this point it is still possible for the upgrade handshake to fail and for the channel
 > to remain in the pre-upgraded state. 
+
+## Upgrading channels with the CLI
+
+A new cli has been added which enables either
+    - submitting a governance proposal which contains a `MsgChanUpgradeInit` for every channel to be upgraded.
+    - generation a `proposal.json` file which contains the proposal contents to be edited/submitted at a later date.
+
+The following example, would submit a governance proposal with the specified deposit, title and summary which would
+contain a `MsgChanUpgradeInit` for all `OPEN` channels whose port matches the regular expression `transfer`.
```

Comment:
```suggestion
The following example, would submit a governance proposal with the specified deposit, title and summary while also containing a `MsgChannelUpgradeInit` for all `OPEN` channels whose port matches the regular expression `transfer`.
```

Reply:
i kind of prefer @chatton's version tbh bc the proposal is there to contain (approve) the `MsgChannelUpgradeInit`, but it's a super nit really.

Reply:
will leave as is if you guys are okay with it then?

Reply:
ah, maybe just the `MsgChannelUpgradeInit` as opposed to `MsgChanUpgradeInit`? (git diff isn't particularly good at showing the specific diffs).

Wording is perfectly fine by me either way!

Reply:
ah missed the typo, thanks @DimitrisJim